guides/doc-Upgrading_and_Updating/topics/migrating_pulp_content.adoc

```diff
@@ -0,0 +1,72 @@
+[[migrating_pulp_content]]
+
+= Migrating Content to Pulp 3
+
+The time to prepare content for Pulp 3 depends on the amount of content and the number of content views.
+For large systems, this can mean several days of downtime.
+To prevent this, {Team} recommends that you pre-migrate Pulp content while running the latest version of {ProjectServer} {ProjectVersionPrevious}.
```

Comment:
```suggestion
To prevent this, {Team} recommends that you pre-migrate Pulp content while running {ProjectServer} {ProjectVersionPrevious}.
```

I feel like "latest" does not go together with "previous", or am I missunderstanding?

Reply:
This is the way its done. Then when we move from one version to the next all that need to be updated are the variable values and the version numbers are updated in all the documents. If at 6.10 we want to say before migrating you must have updated to the latest version of 6.9 beforehand, then 6.9 is {ProjectPreviousversion}.

Reply:
Thanks for clarifying!